### PR TITLE
Feat/usdc price

### PR DESCRIPTION
**Pull request by kenta-elys**
- [x] Add a special endpoint for querying usdc price on oracle.

### CODE REVIEW

File: src/msg/query_msg.rs [hunk 1]
```diff
@@ -48,4 +50,6 @@ pub enum QueryMsg {
     GetCommitments { delegator_addr: String },
     #[returns(QueryEarnPoolResponse)]
     GetLiquidityPools { pool_ids: Option<Vec<u64>>, filter_type: FilterType, pagination: Option<PageRequest> },
+    #[returns(Decimal)]
```
Comment (politeWall) [inline src/msg/query_msg.rs:53]: it would be better to return a Response msg

Comment (kenta-elys) [reply src/msg/query_msg.rs:53]: Got it

File: src/action/query/earn/get_usdc_price.rs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+use super::*;
+use cosmwasm_std::{Decimal, Uint128};
+use crate::bindings::{query::ElysQuery, querier::ElysQuerier};
+use crate::types::ElysDenom;
+
+pub fn get_usdc_price(deps: Deps<ElysQuery>) -> Result<Decimal, ContractError> {
+    let querier = ElysQuerier::new(&deps.querier);
+    let usdc_oracle_price = querier.get_oracle_price(ElysDenom::USDC.as_str().to_string(), "".to_string(), 0)?;
+    let usdc_usd_price = usdc_oracle_price.price.price.checked_div(Decimal::from_atomics(Uint128::new(1000000), 0).unwrap()).unwrap();
+    Ok(usdc_usd_price)
```
[the remark this answers is not in the document]
Comment (kenta-elys) [reply src/action/query/earn/get_usdc_price.rs:10]: Got it

File: src/msg/query_msg.rs (hunk 1 above)
Comment (politeWall) [reply src/msg/query_msg.rs:53]: you forgot to change it here

**Review by politeWall — APPROVED** [review]

**Pull request merged**